src/modules/users/pages/RegisterUser/components/RegisterUserForm/components/PersonalInfoForm/PersonalInfoForm.tsx

```diff
@@ -76,6 +77,7 @@ const PersonalInfoForm: React.FC<PersonalInfoFormProps> = ({ control }) => {
               defaultValue=""
               rules={{
                 required: 'Data de nascimento do usuário é obrigatória',
+                validate: (date: string) => DateUtils.validateDate(date),
```

Comment:
Ele deixa ser escrito, mas se caso for submetido, estoura um erro na tela. É isso?

Reply:
Isso